### DISCUSSION TITLE
Doamin "githubhelp.com" belong to Github?

### DISCUSSION THREAD

**Discussion by cutescorpion** [community/community#23581] [New to GitHub]
<p>Just confuse for me , Doamin “<a href="http://githubhelp.com" rel="noopener nofollow ugc">githubhelp.com</a>” belong to Github?</p>

**Comment by mpboom** [comment, marked as answer]
<p>Hi <a class="mention" href="/cutescorpion">@cutescorpion</a> and welcome to the forum!</p>
<p>No, <a href="http://githubhelp.com" rel="noopener nofollow ugc">githubhelp.com</a> does is <em>not</em> affiliated with GitHub in any way.</p>

**Comment by cutescorpion** [comment]
<p>Thanks your reply , let me follow correctly(maybe).</p>
